### PR TITLE
 connlist updates: fix newline char + use context.WithTimeOut

### PR DESCRIPTION
**Pull request by adisos**
Signed-off-by: adisos <adisos@il.ibm.com>

Remaining items from issue #50

### CODE REVIEW

**Review by zivnevo — DISMISSED** [review]

File: cmd/netpolicy/cmd/evaluate.go [hunk 1]
```diff
@@ -127,8 +128,13 @@ var evaluateCmd = &cobra.Command{
 		} else {
 			// get relevant resources from k8s live cluster
 			var err error
+			const ctxTimeoutSeconds = 3
+			ctx := context.Background()
+			ctx, cancel := context.WithTimeout(ctx, ctxTimeoutSeconds*time.Second)
```
Comment (elevran) [inline cmd/netpolicy/cmd/evaluate.go:133]: `ctx` is being overridden.  Can refer to `context.Background` in the call when inializing the actual context

File: pkg/netpol/connlist/connlist.go [hunk 2]
```diff
@@ -110,8 +112,12 @@ func FromK8sCluster(clientset *kubernetes.Clientset) ([]Peer2PeerConnection, err
 
 	// get all resources from k8s cluster
 
+	ctx := context.Background()
```
Comment (elevran) [inline pkg/netpol/connlist/connlist.go:115]: Reused var name in next line. No need to create it separately

File: pkg/netpol/connlist/connlist.go [hunk 3]
```diff
@@ -188,7 +194,8 @@ func ConnectionsListToString(conns []Peer2PeerConnection) string {
 		connLines[i] = conns[i].String()
 	}
 	sort.Strings(connLines)
-	return strings.Join(connLines, "\n")
+	newlineChar := fmt.Sprintln("")
```
Comment (elevran) [inline pkg/netpol/connlist/connlist.go:197]: Not sure I understand the problem with using `\n`
That is a line break in all platforms but Windows

**Review by elevran — CHANGES_REQUESTED** [review]

File: pkg/netpol/connlist/connlist.go (hunk 3 above)
Comment (elevran) [reply pkg/netpol/connlist/connlist.go:196]: Println is better if you care about Windows as a runtime platform

**Pull request merged**